scala-package/core/src/main/scala/org/apache/mxnet/javaapi/DType.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.mxnet.javaapi
```

Comment:
cosmetic: could you please see if you can instead use `org.apache.mxnet.java` instead of `org.apache.mxnet.javaapi`

Reply:
I'd prefer to use that name also, the concern we had was about users potentially having namespace issues if we ended the package as .java. I'll do some testing to see if this was a valid concern.

Reply:
Ah, now I remember what the problem was with using java. When @lanking520 was working on the NDArray macro code the name cause conflicts with things like java.lang.Float.

I'll spend some time today trying to get this to work as I'd much prefer using the name java to javaapi.

Reply:
@nswamy that namespace would break many import statement in all package under the name `org.apache.mxnet`. All Scala code trying to fetch java library such as `java.io` would break due to this confusion.

Reply:
So it appears that in Scala that all subpackages are in the namespace of the package. So in this case, the package org.apache.mxnet.java would be in the namespace for all of org.apache.mxnet. This leads to obvious issues when trying to call anything from the java library because all calls to java.foo.bar would be looking in the org.apache.mxnet.java namespace instead of the java library namespace.

You could use _root_.java.foo.bar to get around this but I don't believe it's worth the headache.

Reply:
@lanking520 @andrewfayres thanks for checking and explaining the reason to name `javaapi`. This looks good to me.